src/Core/src/Platform/Windows/MauiWinUIWindow.cs

```diff
@@ -75,6 +75,13 @@ protected virtual void OnActivated(object sender, UI.Xaml.WindowActivatedEventAr
 				else
 					_enableResumeEvent = true;
 			}
+			else if (args.WindowActivationState == UI.Xaml.WindowActivationState.Deactivated &&
+				!_isActivated)
+			{
+				// Don't invoke deactivated event if we're not activated. It's possible we can
+				// recieve this event twice if we start more than one process at a time
```

Comment:
Why would two processes be using the same MauiWinUIWindow?

Reply:
Ah, they're not the same `MauiWinUIWindow`: starting _any_ process that has a window will send a window message back to the window of the process that started it:
![image](https://github.com/dotnet/maui/assets/890772/485215e3-f1a7-444a-9440-089d85193081)